src/gui/keymap.cpp

```diff
@@ -108,10 +108,10 @@ static const KeyPatch patchSE[] = {
     {"angle", "<", "bslash_iso"}, {"minus", "-", "slash"},
 };
 
-static const KeyPatch patchJP[] = {
+/*static const KeyPatch patchJP[] = {
     /*{"lbrace", "[", "rbrace"},
```

Comment:
Apologies, one more thing I noticed just now. Please remove this `/*` that's inside the comment.